ethstats/ethstats.go

```diff
@@ -545,7 +544,13 @@ func (s *Service) readLoop(conn *websocket.Conn) {
 			log.Info("Ping from websocket server", "msg", msg)
 			// Primus server might want to have a pong or it closes the connection
 			var serverTime = fmt.Sprintf("primus::pong::%d", time.Now().UnixNano()/int64(time.Millisecond))
-			conn.WriteJSON(serverTime)
+			var pong = map[string]interface{}{
```

Comment:
Is this coming from the Primus protocol?

Reply:
not this change is coming from the linter, writeJSON should write a JSON structure. I was never able to trigger this piece of code so it might make no difference.

Reply:
from https://github.com/primus/primus#protocol
```
primus::pong::<ping> client -> server, The pong is the response to the ping packet. It echoes back the exact value that it received.
```

So the packet that Primus expects should be of the form `primus::pong::<ping>`, which I'm not sure now will be the case. Can you verify?

Reply:
if this is the case WriteJSON is not the right method to do this. i will check

Reply:
i changed it to write message